### DISCUSSION TITLE
Unlock all achievements from home screen

### DISCUSSION THREAD

**Discussion by ghost** [syntax-tm/SteamAchievementManager#24] [Ideas]
Add an unlock all achievements button in the right click context menu in the home screen (in 0.7.x), also an option to unlock every achievement in every game with 1 click somewhere in the settings 😁

**Comment by syntax-tm** [comment]
It's doable, but not super cleanly. It's going to have to launch another process so that it can initialize the Steam client as that app since a process can only initialize as one app for that processes lifetime and launching SAM viewing your library that main process' AppId is set to `0`.

I'll see if I can move some stuff around and add the argument to pass in. All of the other stuff with managing apps and detecting that process and when it exits and stuff is the potentially more complicated piece.
